controllers/tests/controller_server/host_definer/utils/k8s_manifests_utils.py

```diff
@@ -0,0 +1,114 @@
+import controllers.tests.controller_server.host_definer.settings as settings
+from controllers.servers.settings import (SECRET_ARRAY_PARAMETER,
+                                          SECRET_PASSWORD_PARAMETER,
+                                          SECRET_USERNAME_PARAMETER)
+
+
+def get_k8s_csi_node_manifest(csi_provisioner):
```

Comment:
why provisioner in DRIVERS and is it the object or just a name?

Reply:
it is just the provisioner name, it should be under `drivers`, because the `drivers` has a list of all CSI providers on the node

Reply:
can we change it to `csi_provisioner_name` or something like it?